app/backend/wells/management/commands/export.py

```diff
@@ -238,7 +238,7 @@ def generate_files(self, zip_filename, spreadsheet_filename):
         ########
         # SCREEN
         ########
-        screen_sql = ("""select screen.well_tag_number, screen_from, screen_to, internal_diameter,
+        screen_sql = ("""select screen.well_tag_number, screen_from, screen_to, diameter,
```

Comment:
Shouldn't this be: screen_diameter?
```suggestion
        screen_sql = ("""select screen.well_tag_number, screen_from, screen_to, screen_diameter,
```

Reply:
yes good catch ty